docs/administrator-docs/building.md

```diff
@@ -31,6 +31,9 @@ All package builds begin with the first two steps:
     `./build --list-platforms`
     `./build <platform> all`
 
+4. Run the executable:
+    `./Jellyfin.Server/bin/Release/netcoreapp<dotnetcore-version>/<platform>/jellyfin`
```

Comment:
I actually think the build scripts output to a specific directory, but I'm not sure which one. If this is the case we should update to the proper directory, otherwise this is fine. @joshuaboniface might know.